kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerImpl.java

```diff
@@ -1108,6 +1108,18 @@ protected KieServerController getController() {
         return controller;
     }
 
+    protected void notifyStatusToControllers() {
+        new Thread(() -> {
+            try {
+                Thread.sleep(Long.getLong(KieServerConstants.KIE_SERVER_NOTIFY_UPDATES_TO_CONTROLLERS_DELAY, 1000L));
+            } catch (InterruptedException e) {
+                // do nothing
+            } finally {
+                getDefaultController().update(getInternalServerState());
```

Comment:
Maybe a nitpick, but why is the update executed even in case of an exception?
IMHO in case of InterruptedException the thread should be simply terminated, not trying to trigger functionality not related to teardown.

Reply:
make sense